documentation/modules/overview/con-configuration-points-configmaps.adoc

```diff
@@ -0,0 +1,78 @@
+// This module is included in:
+//
+// assembly-overview.adoc
+
+[id="configuration-points-configmaps-{context}"]
+= Using ConfigMaps to add configuration 
+
+[role="_abstract"]
+Use `ConfigMap` resources to add configuration to your Strimzi deployment.
+ConfigMaps store key-value pairs and can be a more secure means of storing configuration data, particularly confidential data like passwords.  
+Configuration data added to ConfigMaps is maintained in one place and can be reused amongst components.
```

Comment:
Does this mean that one ConfigMap is used by many components? I thought there was one ConfigMap per component

Reply:
We suggest that you can reuse a ConfigMap for logging, at least. This is what we have in [Logging configuration](https://strimzi.io/docs/operators/in-development/configuring.html#external-logging_str):

 _The advantages of using a ConfigMap are that the logging properties are maintained in one place and are accessible to more than one resource. You can also reuse the ConfigMap for more than one resource._